MATLAB/config.m

```diff
@@ -70,6 +70,7 @@
 % Torque on fuel/difference in angular rates in eci Nm/(rad/s).
 const.ATTITUDE_PD_KP = 75e-4; % imported from simulink
 const.ATTITUDE_PD_KD = 32.5e-4; % imported from simulink
+const.ATTITUDE_PD_derivative_buffer_size=1;
```

Comment:
Will we need this for the flight algorithm you think? We're already buffering `w_body` in the estimation step.

Reply:
I'm not sure, I wanted to have way to point even if `quat_body_eci` was NaN. Before, the derivative term was just the gyro, but I wanted to see what would happen if I just did finite differences.

Reply:
Though I guess this basically doing the same thing as estimating `w_body`

Reply:
Ah okay. But in terms of the sorta version 1.0 of the GNC algorithms we're good keeping the finite difference estimation in the estimator and no buffer in the controller?